extension/js/common/composer.ts

```diff
@@ -1072,7 +1074,10 @@ export class Composer {
       Xss.sanitizeAppend(inputAddrContainer, selectElHtml);
       inputAddrContainer.find('#input_from_settings').click(Ui.event.handle(() => this.app.renderSendingAddrDialog(), this.getErrHandlers(`open sending address dialog`)));
       const fmtOpt = (addr: string) => `<option value="${Xss.escape(addr)}" ${this.getSender() === addr ? 'selected' : ''}>${Xss.escape(addr)}</option>`;
-      Xss.sanitizeAppend(inputAddrContainer.find('#input_from'), addresses.map(fmtOpt).join('')).change(() => this.composerContacts.updatePubkeyIcon());
+      emailAliases = emailAliases.sort((a, b) => {
+        return (sendAs[a].isDefault === sendAs[b].isDefault) ? 0 : sendAs[a].isDefault ? -1 : 1;
```

Comment:
This looks good. Maybe we can completely remove the "choose order of aliases" screen, since they can do this in Gmail settings?

Reply:
In my opinion, it would be better to leave it as is. The way users can change email alias using FlowCrypt is much easier. When I tried to change my alias first time in Gmail I spent too much time to find the tab with email aliases I was looking for.
And maybe, some users want to have different email aliases in Gmail and in FlowCrpypt, so I would recommend you to leave this screen.

Reply:
There is some truth to it, but presumably these people also use normal unencrypted email, so hopefully they already have gone through the pain of configuring it.

I think the best possible scenario is that the user would never have to configure anything in FlowCrypt and it would just work as expected. That's not realistic, but this update would push us a bit closer in that direction. 

One worry is that we may end up needing this for Microsoft accounts.. but regardless I would rather have it removed if it's not necessary. 

FlowCrypt is becoming a bit of an effort to maintain, so less code is better.

Reply:
Okay, that makes sense,

Then I will remove it